internal/editor/editor.go

```diff
@@ -128,3 +86,25 @@ func Invoke(ctx context.Context, editor string, content []byte) ([]byte, error)
 
 	return nContent, nil
 }
+
+func vimOptions(editor string) []string {
+	if editor != "vi" && editor != "vim" && editor != "neovim" {
+		return []string{}
+	}
+
+	path := "/dev/shm/gopass*"
+	if runtime.GOOS == "darwin" {
+		path = "/private/**/gopass**"
+	}
+	viminfo := `viminfo=""`
+	if editor == "neovim" {
+		viminfo = `shada=""`
+	}
+
+	return []string{
+		"-i", "NONE", // disable viminfo
+		"-n", // disable swap
+		"-c",
+		fmt.Sprintf("autocmd BufNewFile,BufRead %s setlocal noswapfile nobackup noundofile %s", path, viminfo),
```

Comment:
I guess we might want to document the fact that we're setting these somewhere in the documentation, no?